app/models/companies_import.rb

```diff
@@ -0,0 +1,63 @@
+class CompaniesImport
+  include ActiveModel::Model
```

Comment:
Please check if all methods should be public, if no, hide them bellow 'private' keyword